packages/medusa/src/repositories/money-amount.ts

```diff
@@ -256,6 +345,60 @@ export const MoneyAmountRepository = dataSource
       return [result[0][variant_id], result[1]]
     },
 
+    async findCurrencyMoneyAmounts(
+      where: { variant_id: string; currency_code: string }[]
+    ) {
+      const qb = this.createQueryBuilder("ma")
+        .leftJoin(
+          "product_variant_money_amount",
+          "mav",
+          "mav.money_amount_id = ma.id"
+        )
+        .addSelect("mav.variant_id", "variant_id")
+
+      where.forEach((w, i) =>
```

Comment:
suggestion: Can we use better wording than w, i. maybe something like clauses, index?

Reply:
in general if we could just avoid one letter naming as much as possible unless it is an inline code?